### PR TITLE
Migrate command `updateledgers`

### PR DESCRIPTION
**Pull request by zymap**
Descriptions of the changes in this PR:

#2057 

```
Update bookie id in ledgers (this may take a long time).

Usage:  bkctl ledger update [flags]

Flags:

    -host, --hostname
        Expects configuration useHostNameAsBookieID=true as the optin value

    -l, --limit
        Maximum number of ledgers of ledgers to update (default: no limit)

    -p, --printprogress
        Print messages on every configured seconds if verbose turned on
        (default: 10 secs)

    -v, --verbose
        Print status of the ledger updation (default: false)

    -s, updatepersec
        Number of ledgers updating per second (default: 5 per sec)


    -h, --help
        Display help information
```

### CODE REVIEW

**Comment by jiazhai** [comment]
run integration tests
rebuild java11
run bookkeeper-server remaining tests

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/UpdateLedgersCommand.java [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+ * Command to update ledger command.
+ */
+public class UpdateLedgersCommand extends BookieCommand<UpdateLedgersCommand.UpdataLedgersFlags> {
+
+    static final Logger LOG = LoggerFactory.getLogger(UpdateLedgersCommand.class);
+
+    private static final String NAME = "update";
+    private static final String DESC = "Update bookie id in ledgers (this may take a long time).";
+
+//    private static final Long
```
Comment (sijie) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/UpdateLedgersCommand.java]: remove this?

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/UpdateLedgersCommand.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+     */
+    @Accessors(fluent = true)
+    @Setter
+    public static class UpdataLedgersFlags extends CliFlags {
+
+        @Parameter(names = { "-host", "--hostname" },
+            description = "Expects configuration useHostNameAsBookieID=true as the optin value")
+        private boolean hostname;
+
+        @Parameter(names = { "-s", "updatepersec" },
```
Comment (sijie) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/tools/cli/commands/client/UpdateLedgersCommand.java]: --updatepersec

File: tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/LedgerCommandGroup.java [hunk 3]
```diff
@@ -42,6 +43,7 @@
         .withCategory(CATEGORY_LEDGER_SERVICE)
         .addCommand(new SimpleTestCommand())
         .addCommand(new DeleteLedgerCommand())
+        .addCommand(new UpdateLedgersCommand())
```
Comment (sijie) [inline tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/LedgerCommandGroup.java]: I think this command is converting  the bookie id between ip address and hostname. it is required to run on a specific bookie. I would suggest adding this command to `BookieCommandGroup` and call this command as `flip-bookie-id` or `change-bookie-id`.

Comment (zymap) [reply tools/ledger/src/main/java/org/apache/bookkeeper/tools/cli/commands/LedgerCommandGroup.java]: ok, I will fix it later

**Comment by zymap** [comment]
run pr validation

**Comment by zymap** [comment]
run bookkeeper-server replication tests

**Pull request merged**